Tests/ContentfulTests/QueryTests.swift

```diff
@@ -528,29 +566,38 @@ class QueryTests: XCTestCase {
     func testFetchEntriesWithLocationProximitySearch() {
         let expectation = self.expectation(description: "Location proximity search")
 
-        let query = QueryOn<City>(where: "fields.center", .isNear(CLLocationCoordinate2D(latitude: 38, longitude: -122)))
+        let query = QueryOn<City>(where: "fields.center", .isNear(Location(latitude: 38, longitude: -122)))
 
-        QueryTests.client.fetchMappedEntries(with: query).then { citiesResponse in
+        QueryTests.client.fetchMappedEntries(with: query) { result in
+            switch result {
+            case .success(let citiesResponse):
             let cities = citiesResponse.items
```

Comment:
Looks like you missed an indentation here.

Reply:
👍